bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -111,7 +111,7 @@ struct Config {
   uint32_t refreshTimerMilli = 300; // ms
   uint32_t checkpointSummariesRetransmissionTimeoutMilli = 2500; // ms
   uint32_t maxAcceptableMsgDelayMilli = 60000; // 1 minute
-  uint32_t sourceReplicaReplacementTimeoutMilli = 600000; // 10 minutes
+  uint32_t sourceReplicaReplacementTimeoutMilli = 1000; // 1 second

```

Comment:
This mitigates the problem that when replicas communicate via UDP, the fetcher cannot know the source replica is down. Changing the source replica frequently solves this problem and, IMHO does not introduce overhead.